.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
FWIW, on my laptop, the difference between ignoring .git and not ignoring it is 0.10 seconds on average.
My 2 cents is that it's not worth the effort. Speed freaks (like me) either way tend to build natively instead of inside a container.